test/Swashbuckle.AspNetCore.Cli.Test/ToolTests.cs

```diff
@@ -143,5 +166,20 @@ private static JsonDocument RunApplication(Func<string, string[]> setup)
             string json = File.ReadAllText(outputPath);
             return JsonDocument.Parse(json);
         }
+
+        private static string GenerateRandomString(int length)
+        {
+            const string chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+            StringBuilder result = new StringBuilder(length);
+            Random random = new Random();
+
+            for (int i = 0; i < length; i++)
+            {
+                result.Append(chars[random.Next(chars.Length)]);
+            }
```

Comment:
```suggestion
            const string Choices = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
            var result = new StringBuilder(length);

            for (int i = 0; i < length; i++)
            {
                result.Append(Choices[Random.Shared.Next(Choices.Length)]);
            }
```

Shame we can't just use this which would work in .NET 8+:

```csharp
const string Choices = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
var items = Random.Shared.GetItems<char>(Choices, length);
return new string(items);
```